app/main.py

```diff
@@ -1,13 +1,22 @@
+from idlelib.editor import keynames
```

Comment:
The import `from idlelib.editor import keynames` seems unnecessary as `keynames` is not used anywhere in the code. Consider removing it to clean up the imports.